### DISCUSSION TITLE
Restore default sorting of IP addresses

### DISCUSSION THREAD

**Discussion by dreng** [netbox-community/netbox#9295] [Q&A]
I'm not sure if this is a bug or if I'm missing something. If I navigate to https://mydomain/ipam/prefixes/1/ip-addresses/ I get the default and expected sorting like this:
![grafik](https://user-images.githubusercontent.com/20901110/166687605-2d567458-1258-4f61-b66d-8fa6a95bd1e7.png)

None of the column names has a sort icon like "^" at that moment.

Then when I click a column name to sort, there is no obvious way back to the default sort. The corresponding URL would be e.g. https://mydomain/ipam/prefixes/1/ip-addresses/?sort=address. Since Netbox remembers the sorting, it does not help to navigate the view back and forth.

I figured out a workaround by manually editing the URL and entering a value after "sort" that does not correspond to any column. But IMO there should be a button or something to reset to the default view.

Any thoughts?

**Comment by shatt79** [comment]
The only way that I know of to restore the sorting to default is by going to your profile, clicking the preferences tab, and clearing the sort preferences for the given table.  I desperately wish there was a "reset sorting" button or something on the actual table's page.  We are getting ready to deploy NB in our organization and I feel that this is going to be a pain point for users.

**Reply by jeremystretch** [reply]
>  I desperately wish there was a "reset sorting" button or something on the actual table's page.

Sounds like it would be worth a feature request.

**Reply by dreng** [reply]
Seems that this has been done 2 days ago in #9328. Although it is declared as a bug.
